fabtests/pytest/conftest.py

```diff
@@ -1,85 +1,75 @@
 import pytest
 
+def get_option_longform(option_name, option_params):
+    '''
+        get the long form command line option name of an option
+    '''
+    return option_params.get("longform", "--" + option_name.replace("_", "-"))
+
 def pytest_addoption(parser):
+    import yaml, builtins
+
     parser.addoption("--provider", dest="provider", help="libfabric provider")
```

Comment:
Any reason `--provider`, `--client-id` and `--service-id` are not put into `options.yaml`?

Reply:
They are different for pytest and runfabtests.py. For `runfabtests.py`, `provider`, `client_id` and `server_id` are not options but positional arguments, e.g. they are the 1st, 2nd and 3rd command line arguments.

Reply:
Thanks for the clarification.

Details on the comment about the commit message (why guihub doesn't support commenting on commit messages?):

> python's argparse has a count action, which can convert reported
> **occurance** of an argument in to a number. This patch used the

occurrence

> count action to implement verbosity level, which **resulted** **simplied**

resulted in simplified

> implementation 

The second sentence can also use present tense instead of past tense.

Reply:
Thank you! Adjusted the comments as you pointed.